loadout.py

```diff
@@ -22,15 +22,15 @@ def export(data, filename=None):
     # Look for last ship of this type
     ship = companion.ship_file_name(data['ship'].get('shipName'), data['ship']['name'])
     regexp = re.compile(re.escape(ship) + '\.\d\d\d\d\-\d\d\-\d\dT\d\d\.\d\d\.\d\d\.txt')
-    oldfiles = sorted([x for x in os.listdir(config.get('outdir')) if regexp.match(x)])
+    oldfiles = sorted([x for x in os.listdir(config.get_str('outdir')) if regexp.match(x)])
     if oldfiles:
-        with open(join(config.get('outdir'), oldfiles[-1]), 'rU') as h:
+        with open(join(config.get_str('outdir'), oldfiles[-1]), 'rU') as h:
             if h.read() == string:
                 return	# same as last time - don't write
 
-    querytime = config.getint('querytime') or int(time.time())
+    querytime = config.get_int('querytime') or int(time.time())
```

Comment:
Use the default in the get_int() call here ?

Reply:
Yep!